src/core_atmosphere/physics/mpas_atmphys_driver.F

```diff
@@ -196,20 +195,20 @@ subroutine physics_driver(domain,itimestep,xtime_s)
     call mpas_pool_get_dimension(block%dimensions,'cellSolveThreadStart',cellSolveThreadStart)
     call mpas_pool_get_dimension(block%dimensions,'cellSolveThreadEnd',cellSolveThreadEnd)
 
-    call mpas_log_write( 'allocate arrays shared by all physics parameterizations:')
+    !allocate arrays shared by all physics parameterizations:
     call allocate_forall_physics(block%configs)
 
     !physics prep step:
     time_lev = 1
-    call mpas_log_write( 'MPAS to physics' )
+
```

Comment:
@joeolson42 I think it is nice to have these print statements while debugging the code. Could we keep these lines with mpas_log_write?

Reply:
To match NCAR's code. All of your questions have the same answer. Do we want to match NCAR's code or not?